### ISSUE TITLE
Can't able to run ifconfig command

### ISSUE THREAD

**Issue by Sam19004** [termux/termux-packages#12758]
### Problem description

While running ifconfig command it gives error 
cannot open /proc/net/dev (Permission denied)

### What steps will reproduce the bug?

Ifconfig

### System information

termux-info:

```
Warning: cannot open /proc/net/dev (Permission denied). Limited output

```

**Comment by Grimler91** [comment]
Requires root, nothing we can do about this

**Comment by Leif-W** [comment]
I hit this issue with limited ifconfig output.

TL;DR Try `ip` (`pkg i iproute2`).  It's newer, and *may* work.

In #5188, `ip addr` is mentioned as an alternative to `ifconfig -a` that uses the NetLink Linux API, which runs in the kernel (as root?) and may (or may not) give access to desired info or functionality.

In #5044 A link to the [Android 10 Privacy Changes] (https://developer.android.com/about/versions/10/privacy/changes) mentions these changes and suggests using high level Android/Java APIs (may or may not be helpful).

Here's a discussion of [`ifconfig` vs `ip`](https://www.redhat.com/sysadmin/ifconfig-vs-ip).

I'm wondering if anything can be done:

- patch ipconfig to use netlink
- ipconfig as shell script wrapper for ip
- add warning to ipconfig to inform user of ip
- inform ipconfig related issue to use ip
- make a quick FAQ entry

Any other ideas?  I'd like all, but the lowest cost, highest impact might be to inform users with a 1-liner warning in ifconfig to use ip instead.

**Comment by amalbenny** [comment]
> Requires root, nothing we can do about this

Even after I installed root package through `pkg install root-repo`, problem persist.
